client/me/purchases/vat-info/index.tsx

```diff
@@ -236,7 +238,9 @@ function useDisplayVatNotices( {
 			reduxDispatch( removeNotice( 'vat_info_notice' ) );
 			reduxDispatch(
 				errorNotice(
-					translate( 'Your VAT details are not valid. Please check each field and try again.' ),
```

Comment:
I just noticed that there's another instance of this string in `client/my-sites/checkout/composite-checkout/components/wp-checkout.tsx` that will need to be changed also!